src/platform/osx.m

```diff
@@ -24,6 +24,7 @@
 @import Foundation;
 #include <mach-o/dyld.h>
 #include "platform.h"
+#include "../config.h"
```

Comment:
Whoops. Had that in there when I thought I'd need gGlobalConfig. I'll remove that.